src/simularium/rendering/InstancedFiberShader.ts

```diff
@@ -264,7 +264,30 @@ void createTube (float t, vec2 volume, out vec3 offset, out vec3 normal) {
 
   // compute the TBN matrix
   vec3 T = normalize(next - prev);
-  vec3 B = normalize(cross(T, next + prev));
+  vec3 B = vec3(0,0,0);
+  // if normalize(next-prev) and next+prev are parallel, then B will be zero
```

Comment:
This comment was initially more confusing than helpful, since `B` is initialized to be 0 right above it it seemed strange to reassert that it's zero. But then reading the whole section, I'm still a bit confused. Is it justifying why you initialized B as zero?  I also don't understand this well enough overall to understand the `check`/ "special case for which N and B are not well defined". If possible it would be nice to have clearer explanation

Reply:
no prob, i think i can make it clearer

Reply:
the init to zero was really just a safety thing and has nothing to do with what follows.  I tried to improve the comments

Reply:
thanks! it's a lot clearer to me now